themes/socialbase/src/Plugin/Preprocess/Node.php

```diff
@@ -209,6 +209,15 @@ protected function preprocessElement(Element $element, Variables $variables) {
       unset($variables['content']['links']['#lazy_builder']);
     }
 
+    // A landing page has a different way of determining this.
+    if ($node->getType() === 'landing_page') {
```

Comment:
@rollki It looks like we don't have any other cases with no_image here, so maybe it needs to be added to social_landing_page_preprocess_node() or if it doesn't work - then to social_core_preprocess_node().
Could you check and fix that?

Reply:
Moved, everything works fine after changes.